src/sat_cli.py

```diff
@@ -24,10 +24,10 @@ class sat_cli(GroundStation):
     def run(self):
         while(1):
             inStr = self.inputHandler.getInput("$ ")
-            commandStr = "{}.cli.send_cmd({},{})".format(self.satellite, len(inStr), inStr)
+            commandStr = "{}.cli.send_cmd({},{})".format(self.satellite, len(inStr), inStr)     # CLI is a particular service that isn't supposed to be used by operators (Q: is there a check for that, btw?)
             try:
                 transactObj = self.interactive.getTransactionObject(commandStr, self.networkManager)
-                print(transactObj.execute())
+                print(transactObj.execute())    # Q: why is printing not consistent with cli.py?
```

Comment:
I don't quite understand. Why don't we print both in a user-friendly way or as a plain dictionary? Is it because `cli` and `sat_cli` are expected to be used differently?